src/app/table/table.component.html

```diff
@@ -170,213 +170,214 @@ <h6>Kubernetes {{ kubernetesVersion }}</h6>
                   <hr style="margin-top: 5px;margin-bottom: 5px;">
               </th>
               <th style="padding: 0px;">
-                <p style="font-size:11px; text-align: center;  padding-bottom:10px;width:90%">STATUS</p>
+                <p style="font-size:11px; text-align: center;  padding-bottom:10px;width:100%">STATUS</p>
                 <hr style="margin-top: 5px;margin-bottom: 5px;">
               </th>
             </tr>
           </thead>
          
           <tbody>
-            <tr *ngIf="clusterSetupStatus != 'n/a'" style="border-bottom: none">
+            <tr *ngIf="clusterSetupStatus.jobCount != 0" style="border-bottom: none">
               <td style="padding: 0px;vertical-align: middle;">
-                <h6 (click)="clickit(gitlabPipelineUrl)" class="gitlab_stages">Cluster setup</h6>
+                <h6 (click)="clickit(gitlabPipelineUrl)" class="gitlab_stages">CLUSTER SETUP</h6>
               </td>
-              <td class="gitlab_stage_build_status" style="padding: 0px; pointer-events: all;"><span class="gitlab_stage_primary" style="width: 60px;"
+              <td class="gitlab_stage_build_status" style="padding: 0px; pointer-events: all;"><span class="gitlab_stage_primary"
                   [ngClass]='gitlabStageBuildClass(clusterSetupStatus.status)'>{{ clusterSetupStatus.status }}</span>
               </td> 
-              <td style="padding: 0">
+              <td class="dropData" style="padding: 0;width: 13px;">
                   <button class="btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#collapseOnee" aria-expanded="false" aria-controls="collapseOnee" style="padding: 0px;">
-                    <img style="padding: 0px;height: 10px;" src="/assets/images/icons/down-arrow.png" alt="" >
+                    <i class="fa fa-chevron-down" style="font-size: 13px; color: #2399f1"></i>
                   </button> 
                 </td>
             </tr>
-            <div class="accordion   " id="accordionExample" style="width: 270px">
-                <div id="collapseOnee" class="collapse  " aria-labelledby="headingOne" data-parent="#accordionExample">
+            <tr class="accordion   " id="accordionExample" >
```

Comment:
Replace the content like this "accordionExample" and put some good name related to particular effect